.konflux/next/release-plan.yaml

```diff
@@ -8,29 +8,17 @@ metadata:
   namespace: tekton-ecosystem-tenant
 spec:
   application: tektoncd-triggers-next
-  data:
-    mapping:
-      components:
-        - name: tektoncd-triggers-next-webhook
-          repository: "quay.io/openshift-pipelines/openshift-pipelines-pipelines-triggers-webhook-rhel8"
-          tags: [next]
-        - name: tektoncd-triggers-next-controller
-          repository: "quay.io/openshift-pipeline/openshift-pipelines-pipelines-triggers-controller-rhel8"
-          tags: [next]
-        - name: tektoncd-triggers-next-core-interceptors
-          repository: "quay.io/openshift-pipelines/openshift-pipelines-pipelines-triggers-core-interceptors-rhel8"
-          tags: [next]
-        - name: tektoncd-triggers-next-eventlistenersink
-          repository: "quay.io/openshift-pipelines/openshift-pipelines-pipelines-triggers-eventlistenersink-rhel8"
-          tags: [next]
   tenantPipeline:
+    serviceAccountName: release-registry-openshift-pipelines-dev
     pipelineRef:
       resolver: git
       params:
         - name: url
-          value: "https://github.com/konflux-ci/release-service-catalog.git"
+          value: https://github.com/savitaashture/tenant-pipelines.git 
```

Comment:
we'll probably want to move this in `openshift-pipelines/hack` I think.

Reply:
Yes make sense
will add it there